BepInEx.Patcher/BepInEx.Patcher.csproj

```diff
@@ -39,11 +35,5 @@
   <ItemGroup>
     <EmbeddedResource Include="..\bin\patcher\BepInEx.Bootstrap.dll" />
   </ItemGroup>
-  <ItemGroup>
-    <Folder Include="Properties" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Remove="Properties\AssemblyInfo.cs" />
-  </ItemGroup>
```

Comment:
How is this related to semver change? This should be in a separate PR for better testability and ability to revert it if needed.